arch/arm64/boot/dts/rockchip/rk3588s-nanopi-r6-common.dtsi

```diff
@@ -708,3 +741,10 @@
 	clocks = <&hdptxphy_hdmi_clk0>;
 	clock-names = "hdmi0_phy_pll";
 };
+
+/* @TODO: FE has display_subsystem different:
+&display_subsystem {
+	clocks = <&hdptxphy_hdmi0>, <&hdptxphy_hdmi0>;
+	clock-names = "hdmi0_phy_pll", "hdmi1_phy_pll";
```

Comment:
I think this is only necessary for the boards have 2 HDMI outs

Reply:
Makes sense. I guess FriendlyElec devs are economizing: more devices have 2 HDMI inputs so they rather duplicate/override those that _don't_.